packages/client/components/EmailPasswordAuthForm.tsx

```diff
@@ -206,14 +210,20 @@ const EmailPasswordAuthForm = forwardRef((props: Props, ref: any) => {
     if (isSignin) {
       LoginWithPasswordMutation(
         atmosphere,
-        {email, password, invitationToken},
+        {email, password, invitationToken: invitationToken || '', isInvitation: !!invitationToken},
```

Comment:
i made invitationToken non-null on the server, which means we have to do some stuff to accommodate on the client.